### DISCUSSION TITLE
Inquiry Regarding the License of Version 1.0.1

### DISCUSSION THREAD

**Discussion by ys201810** [RUCAIBox/RecBole#1956] [General]
Dear RecBole Project Team,

First and foremost, I would like to express my gratitude for making such an excellent project as RecBole available to the public. I highly appreciate your work and have found it particularly useful for recommendation system.

I had been aiming to use version 1.0.1 of your project for commercial purposes, under the understanding that it was released under the MIT License. However, I recently noticed that from version 1.1.0 onwards, the license has been changed to 'Academic purposes only'.

Before we continue, I would like to seek your perspective on the following matters:

Are there any concerns or restrictions you have regarding the continued commercial use of version 1.0.1?
Does the addition of "Academic Only" apply only to future versions, or does it also retroactively affect previous versions?
My intention is to respect your copyright and adhere to all legal and ethical standards. If there are any additional procedures you recommend or any opinions you would like to share, please do not hesitate to inform me.

I appreciate you taking the time to consider this matter and would be grateful for any response you can provide. 
I apologize for any inconvenience and appreciate your understanding.

Sincerely Yours.

**Comment by kazuma-jp** [comment]
I was also wondering about the same thing. I would be happy to receive an answer.
